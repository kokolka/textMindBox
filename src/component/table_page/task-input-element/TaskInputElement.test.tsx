// import { create } from "react-test-renderer";


// jest.mock('./TaskInputElement', () => ({
//     TaskInputElement: jest.fn(() => {
//         <div data-testid="TaskInputElement"/>
//     })
// }))


// describe('input', () => {
//     it('проверка рендера поля ввода', () => {
//         const result = create(<TaskInputElement/>);

//         //нужно делать мок и проверять со снепшотом
//         const element = result.root.findAllByType('input');
//         expect(element).toBe('email');
//     })
// })
