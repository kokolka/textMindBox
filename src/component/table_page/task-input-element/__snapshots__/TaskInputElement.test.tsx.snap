// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Тестирование отрисовки блока с кнопками renders with initial state 1`] = `
<div
  className="bg-white shadow-lg rounded-md w-full"
>
  <div
    className="grid grid-cols-8 gap-2  justify-center w-96 border-b-2 border-gray-400 p-5 "
  >
    <button
      onClick={[Function]}
    >
      <svg
        height="24"
        version="1.1"
        viewBox="0 -4.5 20 20"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
        xmlnsXlink="http://www.w3.org/1999/xlink"
      >
        <title>
          arrow_up [#337]
        </title>
        <desc>
          Created with Sketch.
        </desc>
        <defs />
        <g
          fill="none"
          fillRule="evenodd"
          id="Page-1"
          stroke="none"
          strokeWidth="1"
        >
          <g
            fill="#000000"
            id="Dribbble-Light-Preview"
            transform="translate(-260.000000, -6684.000000)"
          >
            <g
              id="icons"
              transform="translate(56.000000, 160.000000)"
            >
              <path
                d="M223.707692,6534.63378 L223.707692,6534.63378 C224.097436,6534.22888 224.097436,6533.57338 223.707692,6533.16951 L215.444127,6524.60657 C214.66364,6523.79781 213.397472,6523.79781 212.616986,6524.60657 L204.29246,6533.23165 C203.906714,6533.6324 203.901717,6534.27962 204.282467,6534.68555 C204.671211,6535.10081 205.31179,6535.10495 205.70653,6534.69695 L213.323521,6526.80297 C213.714264,6526.39807 214.346848,6526.39807 214.737591,6526.80297 L222.294621,6534.63378 C222.684365,6535.03868 223.317949,6535.03868 223.707692,6534.63378"
                id="arrow_up-[#337]"
              />
            </g>
          </g>
        </g>
      </svg>
    </button>
    <div
      className="col-span-7"
    >
      <form
        action="#"
        className="grid gap-2 grid-cols-7 justify-center items-center"
        onReset={[Function]}
        onSubmit={[Function]}
      >
        <div
          className="col-span-5"
        >
          <input
            className="w-full focus:outline-none focus:shadow-outline focus:border-blue-300"
            id="newTask"
            name="newTask"
            onBlur={[Function]}
            onChange={[Function]}
            placeholder="Введите новую задачу"
            type="text"
            value=""
          />
        </div>
        <div
          className="col-span-2"
        >
          <button
            className=" p-1 rounded-md flex justify-center items-center border border-current border-black h-5 hover:bg-gray-300 active:bg-gray-50"
            disabled={false}
            type="submit"
          >
            Добавить
          </button>
        </div>
      </form>
    </div>
  </div>
</div>
`;
